src/generator.rs

```diff
@@ -283,11 +283,15 @@ Requires=systemd-zram-setup@{zram_device}.service
 After=systemd-zram-setup@{zram_device}.service
 
 [Mount]
+Where={mount_point}
 What=/dev/{zram_device}
-Where={mount_point:?}
```

Comment:
Please don't reverse the order here: we generally put What= before Where= in mount units, as this is the order on the `mount` commandline.

This is also doing an unrelated change to the formatting of Where. It's the same change as in #94. But please doing mixing unrelated changes into your pull requests.

Reply:
Errr..... I checked if create a CephFS mount in `/etc/fstab` with `_netdev`, the corresponding file ordered as my changes...

Also another example from my /boot/efi:
```
$ cat /run/systemd/generator/boot-efi.mount 
# Automatically generated by systemd-fstab-generator

[Unit]
Documentation=man:fstab(5) man:systemd-fstab-generator(8)
SourcePath=/etc/fstab
Requires=systemd-fsck@dev-disk-by\x2duuid-667C\x2d52E1.service
After=systemd-fsck@dev-disk-by\x2duuid-667C\x2d52E1.service
After=blockdev@dev-disk-by\x2duuid-667C\x2d52E1.target

[Mount]
Where=/boot/efi
What=/dev/disk/by-uuid/667C-52E1
Type=vfat
Options=umask=0077
```

Reply:
> Please don't reverse the order here: we generally put What= before Where= in mount units, as this is the order on the mount commandline.

Ok agree with you (even systemd-fstab-generator is not working in this way), What -> Where -> Type -> Options also in the same order as `/etc/fstab` which looks much comfortable for me, too.

Reply:
https://github.com/systemd/systemd/pull/20411